pkg/datastore/database.go

```diff
@@ -534,20 +558,11 @@ func dbReadNotifier(notifierName string, fetchRecipients bool) (*Notifier, error
 		return nil, err
 	}
 
-	duration, err :=
-		time.ParseDuration(strings.ToLower(strings.Split(readNotifier.Interval, "T")[1]))
-	if err != nil {
-		log.Fatal(err)
-		return nil, err
-	}
-	//log.Tracef("%#v\n", duration.String())
-	readNotifier.Interval = duration.String()
-
 	if fetchRecipients {
 		err = db.Select(
```

Comment:
this error is never checked, and should this also be part of the tx?
@robske110

Reply:
we should indeed check it. Ignoring an error message there results in returning no recipients for the notifier.

Reply:
solved by #[86fa22e](https://github.com/JM-Lemmi/ical-relay/pull/233/commits/86fa22efdcffd1995e227f362d3fbf99fd77c4eb)